### DISCUSSION TITLE
Add TODO tasks for contributors

### DISCUSSION THREAD

**Discussion by illiafox** [indigo-web/indigo#54] [Q&A]
I believe this framework will become popular in near future, so it would be great to add the `TODO` list with delayed tasks and issues, which can me solved by contributors

**Comment by flrdv** [comment]
> I believe this framework will become popular in near future, so it would be great to add the `TODO` list with delayed tasks and issues, which can me solved by contributors

I have a trello workspace, how do you think, where is it better to store the link?

**Reply by flrdv** [reply]
Decided that leaving a link in README is the best solution

**Reply by CptIdea** [reply]
I can't find it

**Reply by flrdv** [reply]
> I can't find it

Removed it due to inactivity. It was more of a burden.
